src/metrics/meter.rs

```diff
@@ -20,6 +21,13 @@ pub struct MeterSnapshot {
     pub mean: f64,
 }
 
+impl Display for MeterSnapshot {
+    fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
+        write!(f, "{}, {}", self.count, self.mean)
```

Comment:
What is this used for? It seems like different things would want to display this in different ways?